tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/settings.html

```diff
@@ -13,7 +13,7 @@
 
     #contentDiv {
```

Comment:
@kiran11990 use control type class instead of id., ex: use contentBody in this case.